pkg/doc/verifiable/credential.go

```diff
@@ -316,12 +381,133 @@ func NewCredential(dataJSON []byte, opts ...CredentialOpt) (*Credential, error)
 	return cred, nil
 }
 
+// CredentialClaims is JWT Claims extension by Credential.
+type CredentialClaims struct {
+	*jwt.Claims
+
+	Credential *rawCredential `json:"vc,omitempty"`
+}
+
+// RawClaims is used to get raw content of "vc" claim of JWT.
+type RawClaims struct {
+	*jwt.Claims
+
+	Raw map[string]interface{} `json:"vc,omitempty"`
+}
+
+func decodeJws(rawJwt []byte, toVerifyJwtSignature bool, fetcher PublicKeyFetcher) ([]byte, *rawCredential, error) {
+	parsedJwt, err := jwt.ParseSigned(string(rawJwt))
+	if err != nil {
+		return nil, nil, fmt.Errorf("VC is not JWS: %w", err)
+	}
+
+	var jwtPayload CredentialClaims
+	err = parsedJwt.UnsafeClaimsWithoutVerification(&jwtPayload)
```

Comment:
This seems fishy to me (contingent on your answer to my [question below](https://github.com/hyperledger/aries-framework-go/pull/312/files#r327225969)).

Reply:
We deserialize JWT claims without verification in order to get `issuer` claim. It could contain a DID of the issuer which should direct us where to look for the public key needed to make signature verification. We can also use `kid` JOSE header to pick appropriate public key if there are several available in the resolved DID Document.
Look at how it's implemented at **decentralized-identity/interop-project**:
https://github.com/decentralized-identity/interop-project/blob/1e64dcbd3470c703c063a14465c3ac8d2a4585ba/cli-demo/src/index.js#L59